trie-db/src/lookup.rs

```diff
@@ -655,4 +668,133 @@ where
 		}
 		Ok(None)
 	}
+
+	/// Look up the closest merkle value of the provided key.
+	///
+	/// When the provided key leads to a node, then the merkle value of that node
+	/// is returned. However, if the key does not lead to a node, then the closest
+	/// merkle value is returned.
+	fn look_up_merkle_without_cache(
```

Comment:
This should be called `lookup_first_descendent_without_cache` or something like this this.

Reply:
That makes sense, thanks for the suggestion!